### PR TITLE
liquid support

### PR DESCRIPTION
**Pull request by coryodaniel**
Still need to skip subdirs or add `{% raw %}` blocks to templates.

Also closes: https://github.com/massdriver-cloud/massdriver-cli/issues/152

### CODE REVIEW

**Review by akasprzok — APPROVED** [review]
Thanks for taking this on, would've taken me forever w/o any CLI context.

**Comment by akasprzok** [comment]
> Still need to skip subdirs or add `{% raw %}` blocks to templates.

On the massdriver side I currently only render "massdriver.yaml" and "README.md".

Not sure what the best option here is overall.

Raw tags seems "cleaner" from an implementation perspective but does complicate the templates in general.

File: pkg/bundle/generate.go [hunk 1]
```diff
@@ -30,6 +34,7 @@ func Generate(data *template.Data) error {
 	data.RepoName = fmt.Sprintf("massdriver-cloud/%s", data.Name)
 	data.RepoNameEncoded = strings.ReplaceAll(data.RepoName, "/", "%2F")
 
-	templateDir, _ := fs.Sub(fs.FS(templatesFs), path.Join("templates", data.TemplateName))
-	return template.RenderEmbededDirectory(templateDir, data)
+	data.TemplateSource = cache.AppTemplateCacheDir()
+	templatePath := path.Join(data.TemplateSource, data.TemplateName)
+	return template.RenderDirectory(templatePath, data)
```
Comment (coryodaniel) [inline pkg/bundle/generate.go:39]: this is using app rendering now. Added tf as a template there.

File: pkg/bundle/generate_test.go [hunk 2]
```diff
@@ -1,62 +1,3 @@
 package bundle_test
 
-import (
```
Comment (coryodaniel) [inline pkg/bundle/generate_test.go:3]: noop, test dupe of template.Render now.

**Comment by coryodaniel** [comment]
Ended up implementing as templating anything in the root dir of the template, and copying any subdirs. This allows us to use rendering in operator files, etc. note [here](https://github.com/massdriver-cloud/massdriver-cli/pull/168/files#diff-677d343ec8027c2775ceaac5f0ca4bf1e2af203d41c8cdb2d77d8bb32d8e5e78R11)

File: pkg/template/files.go [hunk 3]
```diff
@@ -1,81 +1,24 @@
...
-}
-
-func promptAndWrite(file []byte, data *Data, outputPath string) error {
-	tmpl, errTmpl := template.New("tmpl").Delims(OpenPattern, ClosePattern).Parse(string(file))
-	if errTmpl != nil {
-		return errTmpl
-	}
+	var bindings map[string]interface{}
+	inrec, _ := json.Marshal(data)
+	json.Unmarshal(inrec, &bindings)
```
Comment (coryodaniel) [inline pkg/template/files.go:15]: needed a map string interface, had a struct.

**Review by xpositivityx — APPROVED** [review]

**Pull request merged**